plugins/modules/packaging/language/gem.py

```diff
@@ -62,6 +62,11 @@
       These gems will be independent from the global installed ones.
       Specifying this requires user_install to be false.
     required: false
+  bindir:
+    type: path
+    description:
+    - Install executables into a specific directory.
+    required: false
```

Comment:
```suggestion
    version_added: 3.3.0
```

Reply:
Why removing `required: false` ?

Reply:
It doesn't have any effect, so why not? :)

Reply:
We generally try to avoid redundant statements (at least when adding new things).

Reply:
I'm not sure I understand the redundancy part. Do you mean that required is false by default? If so I'm not sure how I should known about this. Should I remove `require: false` from all options in the docs as we are here?

Reply:
> Do you mean that required is false by default?

Yes it is.

> If so I'm not sure how I should known about this.

You probably could have only known by chance (or because for some reason you found the specific part of the docs which state that - if it is stated somewhere).

> Should I remove require: false from all options in the docs as we are here?

It's better to do that in a separate PR, to keep this PR focused on the feature it implements.